drivers/ethernet/Kconfig.stm32_hal

```diff
@@ -94,3 +97,10 @@ config ETH_STM32_CARRIER_CHECK_RX_IDLE_TIMEOUT_MS
 	  PHY's carrier status is re-evaluated.
 
 endif # ETH_STM32_HAL
+
+config NOCACHE_MEMORY
+	default y if SOC_SERIES_STM32H7X
```

Comment:
`if SOC_SERIES_STM32H7X` is superfluous in this case.

Reply:
> `if SOC_SERIES_STM32H7X` is superfluous in this case.

But then it will be enabled for F4/F7 series also. Would it be okay?

Reply:
Since you mention `depends on SOC_SERIES_STM32H7X` just below, I think we're safe

Reply:
Will be done within next push.